conda_forge_tick/update_upstream_versions.py

```diff
@@ -0,0 +1,169 @@
+import networkx as nx
+import requests
+from pkg_resources import parse_version
+import feedparser
+from .utils import parsed_meta_yaml
+
+import logging
+
+logger = logging.getLogger("conda_forge_tick.update_upstream_versions")
+
+class VersionFromFeed:
```

Comment:
Should these been pulled from the library?

Reply:
I don't understand what you are trying to say here

Reply:
Sorry I had things backwards, it seems there are duplicates between this file and https://github.com/regro/cf-scripts/pull/85/files#diff-94f6e88874a59b9d037066a48e6b6c3fR11